atintegrators/BndStrMPoleSymplectic4RadPass

```diff
@@ -0,0 +1,429 @@
+            Elem->EntranceAngle, Elem->ExitAngle, Elem->X0ref, Elem->ByError,
+            Elem->RefDZ, Elem->FringeInt1, Elem->FringeInt2, Elem->FullGap,
+            Elem->T1, Elem->T2, Elem->R1, Elem->R2, Elem->Energy, num_particles);
+    return Elem;
+}
+
+MODULE_DEF(BndStrMPoleSymplectic4RadPass)      /* Dummy module initialisation */
+
+#endif /*defined(MATLAB_MEX_FILE) || defined(PYAT)*/
+
+#if defined(MATLAB_MEX_FILE)
+void mexFunction(	int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[])
+{
+    if (nrhs == 2) {
+        double irho, flen;
+        double Length, BendingAngle, EntranceAngle, ExitAngle, Gap, Fint1, Fint2, X0ref, ByError, RefDZ, Energy;
+        int MaxOrder, NumIntSteps;
+        double *PolynomA, *PolynomB, *R1, *R2, *T1, *T2;
+        double *r_in;
+        const mxArray *ElemData = prhs[0];
+        int num_particles = mxGetN(prhs[1]);
+        Length=atGetDouble(ElemData,"Length"); check_error();
+        BendingAngle=atGetDouble(ElemData,"BendingAngle"); check_error();
+        EntranceAngle=atGetDouble(ElemData,"EntranceAngle"); check_error();
+        ExitAngle=atGetDouble(ElemData,"ExitAngle"); check_error();
+        PolynomA=atGetDoubleArray(ElemData,"PolynomA"); check_error();
+        PolynomB=atGetDoubleArray(ElemData,"PolynomB"); check_error();
+        MaxOrder=atGetLong(ElemData,"MaxOrder"); check_error();
+        NumIntSteps=atGetLong(ElemData,"NumIntSteps"); check_error();
+        Energy=atGetDouble(ElemData,"Energy"); check_error();
+        /*optional fields*/
+        Gap=atGetOptionalDouble(ElemData,"FullGap", 0); check_error();
+        Fint1=atGetOptionalDouble(ElemData,"FringeInt1", 0); check_error();
+        Fint2=atGetOptionalDouble(ElemData,"FringeInt2", 0); check_error();
+        R1=atGetOptionalDoubleArray(ElemData,"R1"); check_error();
+        R2=atGetOptionalDoubleArray(ElemData,"R2"); check_error();
+        T1=atGetOptionalDoubleArray(ElemData,"T1"); check_error();
+        T2=atGetOptionalDoubleArray(ElemData,"T2"); check_error();
+        X0ref=atGetOptionalDouble(ElemData,"X0ref", 0); check_error();
+        ByError=atGetOptionalDouble(ElemData,"ByError", 0); check_error();
+        RefDZ=atGetOptionalDouble(ElemData,"RefDZ", 0); check_error();
+        irho = BendingAngle/Length;
+        flen = 2.0/irho*sin(BendingAngle/2.0); /* field length */
+        if (mxGetM(prhs[1]) != 6) mexErrMsgIdAndTxt("AT:WrongArg","Second argument must be a 6 x N matrix");
+        /* ALLOCATE memory for the output array of the same size as the input */
+        plhs[0] = mxDuplicateArray(prhs[1]);
+        r_in = mxGetDoubles(plhs[0]);
+        BndStrMPoleSymplectic4RadPass(r_in, flen, irho, PolynomA, PolynomB,
+            MaxOrder, NumIntSteps, EntranceAngle, ExitAngle, X0ref, ByError,
+            RefDZ, Fint1, Fint2, Gap, T1, T2, R1, R2, Energy, num_particles);
+    }
+    else if (nrhs == 0) {
+        /* list of required fields */
+	    plhs[0] = mxCreateCellMatrix(8,1);
```

Comment:
Hello,
it's a small detail, but this should be  plhs[0] = mxCreateCellMatrix(9,1);